Sources/VaporAPNS/Options.swift

```diff
@@ -96,44 +96,39 @@ public struct Options: CustomStringConvertible, NodeInitializable {
     }
 
     
-    public init(node: Node, in context: Context) throws {
+    public init(node: Node) throws {
         if let topic = node["topic"]?.string {
             self.topic = topic
-        }else {
+        } else {
             throw InitializeError.noTopic
```

Comment:
Since you are also cleaning up some things, I suggest removing all these if let statements that just throw in the else part of it. 
```swift
guard let topic = node["topic"]?.string else {
   throw InitializeError.noTopic
}
self.topic = topic
```